benchmark-typecheck

```diff
@@ -54,15 +54,13 @@ processRunner({
       runner.runCmd('packages/pkg1/src/components/component-1', [
         'sed',
         '-i',
-        '""',
```

Comment:
What OS are you running? Could be that this needs a condition

Reply:
I'm on Fedora 36 with sed `4.8`.
Which OS and sed are you working with? I'm guessing Linux as well (as macOS's `sed` doesn't have `-i`), so I'm surprised that this did work for you :grin: